test/unit/unicode_spec.js

```diff
@@ -152,69 +150,8 @@ describe("unicode", function () {
       expect(getUnicodeRangeFor(0x0041)).toEqual(0);
       // fi (Alphabetic Presentation Forms)
       expect(getUnicodeRangeFor(0xfb01)).toEqual(62);
-    });
-
-    it("should not get a Unicode range", function () {
-      expect(getUnicodeRangeFor(0x05ff)).toEqual(-1);
-    });
```

Comment:
Wouldn't it make sense to keep a test-case for the fallback code-path where `-1` is returned?